docs/partials-common/install-golioth-firmware-sdk.md

```diff
@@ -16,7 +16,7 @@ values={[
 
 ```console
 cd ~
-git clone --recursive https://github.com/golioth/golioth-firmware-sdk.git
+git clone https://github.com/golioth/golioth-firmware-sdk.git -b v0.9.0
```

Comment:
hmm why not `--recursive` here?

Reply:
Two lines later we have them run `git submodule update --init --recursive` which seems redundant to me, but I can put it back in. There's no harm in having both as we do now.

Reply:
Can we just do `--recursive` without the `git submodule update` line? Like we do for downloading ESP-IDF.

Reply:
Yeah, will do.